src/main/java/org/apache/mesos/hdfs/HDFSScheduler.java

```diff
@@ -551,12 +568,13 @@ private boolean offerNotEnoughResources(Offer offer, double cpus, int mem) {
   }
 
   private void reconcileTasks(SchedulerDriver driver) {
-    // TODO run this method repeatedly with exponential backoff in the case that it takes time for
+    // TODO (elingg) run this method repeatedly with exponential backoff in the case that it takes
+    // time for
     // different slaves to reregister upon master failover.
```

Comment:
actually... that is the maven formatter that is doing that.   I can stop using that :)  I'll check in a version fixed.